atd-vzd/schema/engineering_areas.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE public.engineering_areas(
+    id integer NOT NULL,
+    eng_area_label text,
```

Comment:
it's not obvious from the toolbox script, but in Moped we have a unique constraint on the council district number in order to prevent dupe inserts. in the same way, i think you probably want to apply a unique index to this eng area label.

Reply:
just took a quick look at that layer. are y'all sure we don't need any of these other columns?

<img width="261" alt="Screenshot 2023-06-06 at 10 58 28 AM" src="https://github.com/cityofaustin/atd-vz-data/assets/14793120/a13afb5a-8874-4803-8eed-d116f55c223c">

Reply:
im not sure about the other columns, afaik this feature is just so that the the vz team can search for fatalities by engineering area/get lists of fatalities for each engineering area.

Reply:
Teams Mary on 6/6/23 about other columns, she said she wanted Council Districts, but not the way they were being aggregated in to the Engineering Areas in the map. Made a separate Issue [#12503](https://github.com/cityofaustin/atd-data-tech/issues/12503) to capture them in the crash export.